modules/products/confluence/main.tf

```diff
@@ -28,3 +28,18 @@ module "database" {
   db_master_username      = var.db_master_username
   db_master_password      = var.db_master_password
 }
+
+module "nfs" {
+  source = "../../AWS/nfs"
+
+  namespace               = var.namespace
+  product                 = local.product_name
+  requests_cpu            = var.nfs_requests_cpu
+  requests_memory         = var.nfs_requests_memory
+  limits_cpu              = var.nfs_limits_cpu
+  limits_memory           = var.nfs_limits_memory
+  capacity                = var.shared_home_size
+  availability_zone       = var.eks.availability_zone
+  shared_home_snapshot_id = var.shared_home_snapshot_id
+  shared_home_size        = var.shared_home_size // This will create PVC for Confluence instance
```

Comment:
Why we have the comment here?

Reply:
This is to distinguish between `shared_home_size` and `capacity`. 
`capacity` is used for creating NFS server PVC, and `shared_home_size` is used for creating Jira & Confluence PVC.

Reply:
Shouldn't be both the same size?

Reply:
I find this confusing, both `capacity` and `shared_home_size` are set using the same value. 
```
capacity = var.shared_home_size
shared_home_size = var.shared_home_size
```

Why do we need both?